src/cunumeric/execution_policy/reduction/scalar_reduction.cuh

```diff
@@ -0,0 +1,63 @@
+#ifndef SRC_CUNUMERIC_EXECUTION_POLICY_REDUCTION_SCALAR_REDUCTION_CUH_
```

Comment:
any reason you prefer this to `#pragma once`? note that we've bee using the pragma in all the other files except for C headers.

Reply:
Force of habit. I will switch it to pragma.